### PR TITLE
Create election mail subscription during import

### PR DESCRIPTION
**Pull request by marlabrizel**
This PR is part of the ongoing TurboVote Classic -> 2.0 user migration work.

Specifically, this piece of the puzzle creates an election mail subscription for any user associated with a partner with an `:unlimited` mailing strategy. Users who were not set up to receive paper mailings in Classic will be imported without a new subscription.
This is done by hitting a new http-api endpoint to query a single partner, which then allows us to get election mail subscription information for a given user.

This PR also covers a bit of opportunistic refactoring to move the initial import data manipulation into its own function.

Tested in Krakenstein.

Addresses [this Pivotal Card](https://www.pivotaltracker.com/story/show/144876025)
[Related PR](https://github.com/democracyworks/partner-http-api/pull/8) in `partner-http-api`

### CODE REVIEW

File: dev-src/user_works/import.clj [hunk 1]
```diff
@@ -79,28 +79,72 @@
...
+          {:status :error
+           :cause user
+           :reason (:body response)}))
+      (catch Exception e
+        {:status :error
+         :cause e
+         :reason (.getMessage e)}))
+    context))
+
+(defn import-election-mail-subscription
```
Comment (ddellacosta) [inline dev-src/user_works/import.clj]: Looking at the error handling in this and `import-user` above, seems like there may be an opportunity to DRY this up somehow with a more generic success-checking/error-handling approach?

File: dev-src/user_works/import.clj [hunk 2]
```diff
@@ -79,28 +79,72 @@
...
-          response (http/post api-url {:content-type :edn
-                                       :body (pr-str user)
-                                       :throw-exceptions false})]
-      (case (:status response)
-        200 {:status :already-exists}
-        201 {:status :success}
-        {:status :error
-         :cause user
-         :reason (:body response)}))
+    {:import-data (edn/read-string line)}
```
Comment (ddellacosta) [inline dev-src/user_works/import.clj]: do I smell a monad

File: dev-src/user_works/import.clj (hunk 1 above)
Comment (marlabrizel) [inline dev-src/user_works/import.clj]: You're absolutely right - this code is pretty wet. I think the inclination here was to let the pattern emerge a bit more in the next story and determine an approach there. How do you feel about that?

File: dev-src/user_works/import.clj (hunk 2 above)
Comment (ddellacosta) [inline dev-src/user_works/import.clj]: nevermind, I couldn't tell you what a monad actually _is_ so 🤣

**Pull request merged**